.github/workflows/admin.yml

```diff
@@ -161,9 +160,6 @@ jobs:
       - name: Run debug commands on failure
         if: ${{ failure() }}
         run: |
-          env
-          go version
-          go env
-          node --version
```

Comment:
why did we drop this line?

Reply:
Because NodeJS is not used anywhere :)

Reply:
I can revert, no prob

Reply:
all commands above uses npx which is a node package, I would also add info about npm, npx and playwright versions

Reply:
Done.